Phaethon/Core/Model/Item.cs

```diff
@@ -27,19 +27,14 @@ public class Item
 
         [Required]
         [DataMember]
-        [DisplayName("Price with tax")]
+        [DisplayName("Incoming price")]
```

Comment:
You can remove all of these, as of the introduction of #47(multilingual webclient), they will not be used.

Reply:
k ill do it

Reply:
done